### ISSUE TITLE
Execution Environment planning

### ISSUE THREAD

**Issue by wateret** [wateret/mengde#35]
Current plan of directory structure for execution

```
mengde
├── game           # executable file
├── liblua.dylib   # lua shared library
├── libutil.dylib  # util shared library
├── {more shared libs}
├── res            # resource directory (currently fonts only)
│   ├── Literation\ Mono\ Powerline.ttf
│   └── {more fonts}
├── sce            # scenario directory
    ├── example
    └── {more scenarios}
```

Issues to resolve

- Make executable `game` finds shared library from in the same directory 
- ...

**Comment by seanshpark** [comment]
I'll work on this. Please assign to me.

**Comment by wateret** [comment]
@seanshpark I can't assign you on the system maybe a permission problem.
Anyway I know you will work on this. Thanks!

**Comment by seanshpark** [comment]
May be add to `Collaborators` :)

**Comment by wateret** [comment]
Resolved with #39 closing
